scripts/verify.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Change the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to `#!/usr/bin/env bash`

See https://github.com/sysprog21/lkmpg/blob/master/.ci/build-n-run.sh